### DISCUSSION TITLE
How to reset wizard after closing modal

### DISCUSSION THREAD

**Discussion by hxnk** [filamentphp/filament#3517] [Help]
Hi!

I have a modal with a wizard that has two steps in it.
When the user closes the modal on the second step, and then opens it again, I would like the wizard to start again at step 1.

Right now all form fields are reset, but the modal opens in step 2.

Anyone know how to accomplish this?

Thanks!

**Comment by danharrin** [comment]
sounds like a bug rather than something to fix on your end? the same probably applies to Tabs

**Reply by hxnk** [reply]
Ah, right.

When I have time I'll take a look to see if I can find the cause...

**Reply by hxnk** [reply]
@danharrin: PR in #3525
